qutip/solver/solver_base.py

```diff
@@ -396,6 +397,29 @@ def add_integrator(cls, integrator, key):
 
         cls._avail_integrators[key] = integrator
 
+    def add_feedback(self, key, type):
```

Comment:
Do we still need these `add_feedback` methods now that they are present on the on `QobjEvo` objects? Perhaps we can just document the supported feedback in the documentation for the solver classes? That probably also makes the existence of feedback more discoverable by users,

Reply:
Having it in `QobjEvo` still feels wrong to me and I prefer using this one.

Reply:
Let me resolve this thread and I'll write a short note in the main conversation instead.